src/components/common/candidate-section/candidate-section.html

```diff
@@ -1,6 +1,6 @@
 <ion-row class="mes-full-width-card" id="candidate-section">
   <ion-col>
-    <h2 id="candidate-name">{{ candidateName }}</h2>
+    <h2 id="candidate-name">Mr Iva Gotta </h2>
```

Comment:
please undo this

Reply:
How the hell did that get into des4...completely baffled.  will revert.

Reply:
reverted